src/CssVariables/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import * as theme from '../theme.js'
+
+/**
+ * Maps over each theme value in a section and converts it to a css custom property (variable)
+ * definition. It uses the export name as a prefix, and suffixes that with the key.
+ */
+const renderThemeSection = ([sectionKey, section]) => {
+    const variables = Object.entries(section)
+
+    return variables.map(([ varKey, varVal ]) => (`--${sectionKey}-${varKey}: ${varVal};`)).join('\n')
+}
+
+/**
+ * Maps over the exported sections of the theme
+ */
+const renderTheme = () => {
+    const sections = Object.entries(theme)
+
+    return sections.map(renderThemeSection).join('\n')
+}
+
+/*
+ * Inject theme values as CSS variables for users that are using vanilla css.
```

Comment:
When using comments, jsdoc block comments would be best (we're moving towards them anyways, see the components in the `next` branch, e. g. the [`Help`](https://github.com/dhis2/ui-core/blob/next/src/Help.js) component)

Reply:
Np, I can move it over to a jsdoc comment. I do find the jsdoc comment on the proptypes a bit redundant by the way. Why are we doing that?

Reply:
For generating markdown prop type tables

Reply:
So those jsdoc comments are used with this lib then: https://www.npmjs.com/package/storybook-addon-react-docgen?

Reply:
No, they're used by [`react-docgen`](https://www.npmjs.com/package/react-docgen) (not the storybook addon, see @varl's PR that has been merged into the `next` branch already):

https://github.com/dhis2/ui-core/pull/397/files?file-filters%5B%5D=.json#diff-b9cfc7f2cdf78a7f4b91a753d10865a2